test/config/100-kafka-persistent-single-2.1.0.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.1.0
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      log.message.format.version: "2.1"
+    storage:
+      type: persistent-claim
+      size: 100Gi
+      deleteClaim: false
+  zookeeper:
```

Comment:
instead of shipping yaml - could we do a script that downloads bits ? or is that problematic ?

I have such a script, therefore asking ;-)

Reply:
The problem I can think about downloading is we will introduce more test flakiness, and if the website is down, we will be blocked as well.
So for now I will prefer shipping over downloading it for every test execution.